client/state/jetpack/credentials/selectors.js

```diff
@@ -0,0 +1,9 @@
+import 'calypso/state/jetpack/init';
+
+export function getAreJetpackCredentialsInvalid( state, siteId, role ) {
```

Comment:
I'd rename this to:

```suggestion
export function areJetpackCredentialsInvalid( state, siteId, role ) {
```

because `getAre` feels counter-intuitive. WDYT?

Reply:
Thanks for this! I'm renaming it.